onnxruntime/contrib_ops/cuda/cuda_contrib_kernels.cc

```diff
@@ -24,6 +24,8 @@ using namespace onnxruntime::common;
 namespace onnxruntime {
 namespace contrib {
 namespace cuda {
+class CUDA_MS_OP_TYPED_CLASS_NAME(1, float, MyTritonKernel);
```

Comment:
Use `#ifdef USE_TRITON_KERNEL` to conditional compile the triton operator.

Reply:
Done